sample-app/src/pages/VerticalSearchPage.tsx

```diff
@@ -68,7 +69,11 @@ export default function VerticalSearchPage(props: {
       universal: {}
     });
     answersActions.setVerticalKey(props.verticalKey);
-    answersActions.executeVerticalQuery();
+    const executeQuery = async () => {
+      const searchIntents = await getSearchIntents(answersActions, false);
```

Comment:
If NEAR_ME stays in the answers state, will this cause searches to always fire an autocomplete request to get the search intents?

Reply:
The intents store in state may not match with the new query that's going to execute next. So the current stored query may have a near me intent but the newly typed input may not, in which case, an autocomplete request is always executed to ensure the intents we are using match with the upcoming input and not what's already executed. tldr, we can't rely on the near me intent store in state

Reply:
Not relying on the near me in state makes sense. It's different than the SDK, however with headless, we can't make the same assumptions